js/latoken.js

```diff
@@ -0,0 +1,715 @@
+                'fetchPartiallyFilledOrders': true,
+                'fetchOrderBook': true,
+                'fetchTicker': true,
+                'fetchTickers': false,
+                'fetchTrades': true,
+                'fetchTransactions': false,
+                'fetchDeposits': false,
+                'fetchWithdrawals': false,
+                'withdraw': false,
+            },
+            'timeframes': {
+                '1m': '1m',
+                '5m': '5m',
+                '15m': '15m',
+                '30m': '30m',
+                '1h': '1h',
+                '3h': '3h',
+                '6h': '6h',
+                '12h': '12h',
+                '1d': '1D',
+                '1w': '7D',
+                '2w': '14D',
+                '1M': '1M',
+            },
+            'urls': {
+                'logo': 'https://user-images.githubusercontent.com/6286552/54519214-0f693600-4977-11e9-9ff4-6ea2f62875b0.png',
+                'api': {
+                    'v1': 'https://api.latoken.com/api/v1',
+                },
+                'www': 'https://www.latoken.com',
+                'doc': [
+                    'https://api.latoken.com/',
+                ],
+            },
+            'api': {
+                'public': {
+                    'get': [
+                        'exchangeInfo/time', 
+                        'exchangeInfo/limits', 
+                        'exchangeInfo/pairs', 
+                        'exchangeInfo/currencies', 
+                        'marketData/orderBook', 
+                        'marketData/trades', 
+                        'marketData/ticker' 
+                    ],
+                },
+                'private': {
+                    'get': [
+                        'account/balances', 
+                        'order/trades', 
+                        'order/status', 
+                        'order/active', 
+                        'order/get_order' 
+                    ],
+                    'post': [
+                        'order/new', 
+                        'order/cancel', 
+                        'order/cancel_all' 
+                    ],
+                    'delete': [
+                        'account/order',  
+                    ],
+                },
+            },
+            'fees': {
+                'trading': {
+                    'tierBased': false,
+                    'percentage': true,
+                    'maker': 0.1 / 100,
+                    'taker': 0.1 / 100,
+                },
+            },
+            'options': {
+                'timeframe': 5 * 1000, // 5 sec, default
+            },
+            'exceptions': undefined,
+        });
+    }
+
+    nonce () {
+        return this.milliseconds ();
+    }
+
+    async currentTime(){
+        const response = await this.publicGetExchangeInfoTime();
+        let time = this.safeString(response, 'time');
+        let timeSeconds = parseInt(response['unixTimeSeconds']);
```

Comment:
`this.safeInt` should be used here instead

Reply:
done